PrebidMobile/PrebidMobileRendering/Prebid/PBMCore/Bid.swift

```diff
@@ -78,7 +78,7 @@ public class Bid: NSObject {
             return false
         }
 
-        for markerKey in ["hb_pb", "hb_bidder", "hb_cache_id"] {
```

Comment:
Let's still verify the `hb_cache_id` for bid requests with the cache. 

If the bid response for the original doesn't contain the `hb_cache_id` it still should mean **no bids**, because the primary ad server SDK won't be able to render it. 

According to the PBS docs, it can happen if something is wrong with the cache instance.